examples/player/player.js

```diff
@@ -22,6 +22,7 @@ elms.forEach(function(elm) {
 var Player = function(playlist) {
   this.playlist = playlist;
   this.index = 0;
+  this.step = this.step.bind(this);
```

Comment:
This works, but it feels weird overwriting this.step with bound this.step rather than storing a separate reference (like this.stepFn or something).